### PR TITLE
Treasury Tipping and FAQ

### LINKED ISSUE #379: Tipping explainer / how-to on Kusama
Could we add something about Tipping on Kusama to the Kusama wiki? https://substrate.dev/rustdocs/master/pallet_treasury/index.html#tipping

### PR DESCRIPTION
**Pull request by lsaether**
closes #379 

also addresses the question @Swader posted on the FAQ PR.

also adds the treasury page to the mirrors and onto the Kusama subwiki.

also updates the governance mirror, apparently.

only changes to the treasury page need to be reviewed.

### CODE REVIEW

File: docs/learn-treasury.md [hunk 1]
```diff
@@ -62,6 +62,100 @@ At this point, a Council member can turn this proposal into a motion to accept,
...
+token value remaining steady. 
+
+Concretely there are a couple on-chain methods that resist this kind of attack.
+One, the council majority may not be the token majority of the chain. This means
+that the token majority could vote to replace the council if they attempted this
+attack - or even reverse the treasury spend. They would do this through a normal
+referendum. Two, there are time delays to treasury spends. They are only enacted
+every spend period. This means that there will be some time to observe this
+attack is taking place. The time delay then allows chain participants time to respond. The response may take the form of governance measures or - in the most
+extream cases a liquidation of their holdings and a migration to a minority fork.
```
Comment (Swader) [inline docs/learn-treasury.md]:
```suggestion
extreme cases a liquidation of their holdings and a migration to a minority fork.
```

**Review by Swader — CHANGES_REQUESTED** [review]
Just a typo. Good explanation on treasury hijack 👌 I'd still worry about an exit scam since by definition the people who do it do not have any reputation worries - the goal is to run away with enough money to never care about such things - but the arguments make sense, especially on the democratic prevention of a hijack once the token majority is a bit more dispersed.

File: docs/learn-treasury.md [hunk 2]
```diff
@@ -62,6 +62,100 @@ At this point, a Council member can turn this proposal into a motion to accept,
...
+One, the council majority may not be the token majority of the chain. This means
+that the token majority could vote to replace the council if they attempted this
+attack - or even reverse the treasury spend. They would do this through a normal
+referendum. Two, there are time delays to treasury spends. They are only enacted
+every spend period. This means that there will be some time to observe this
+attack is taking place. The time delay then allows chain participants time to respond. The response may take the form of governance measures or - in the most
+extreme cases a liquidation of their holdings and a migration to a minority fork.
+However, the possibility of this scenario is quite low.
+
+
```
Comment (lsaether) [inline docs/learn-treasury.md]:
```suggestion
```

**Review by Swader — APPROVED** [review]
LGTM, resolve conflict and let's get this in

**Pull request merged**